molecule/default/converge.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
+  vars:
+    solr_service_name: solr
+    solr_home: /var/solr
+    solr_core_name: test
+    solr_core_conf_source: "./conf"
```

Comment:
@Chithrak07 the playbook is failing to act because this directory is not populated (and doesn't even exist) on the instance. See the readme: 

![image](https://user-images.githubusercontent.com/634971/100927716-1e20b500-349a-11eb-8058-e7d951ebb864.png)

It needs to be an absolute path, relative to the "server" (i.e the subject instance of our molecule tests).

So in `prepare.yml`, you need to add a `synchronize` task to upload the dummy conf source to the "server", and then here in `converge.yml`'s vars, you need to set `solr_core_conf_source` to point at the absolute path you uploaded them to.

In real environments, `solr_core_conf_source` typically gets added to the drupal project's repo, and its' usually available on the server from somewhere such as /home/<owner>/www/<project>/solr_conf/.... or whatever.

Reply:
Done